src/opm/parser/eclipse/EclipseState/Schedule/Action/ActionX.cpp

```diff
@@ -110,5 +111,45 @@ std::vector<DeckKeyword>::const_iterator ActionX::end() const {
     return this->keywords.end();
 }
 
+
+std::vector<std::string> ActionX::keyword_strings() const {
```

Comment:
Could you add a comment explaining what this function does?

Reply:
It returns a `std::vector<std::string>` representation of the keywords which should be applied if the `ACTIONX` keyword evaluates to true. This is purely required to support writing Eclipse formatted restart files.